config/importer.go

```diff
@@ -34,15 +37,19 @@ func (c *Config) String() string {
 
 // Load load the configuration & apply defaults where necessary
 func Load() (*Config, error) {
+	defaultTimeout, _ := time.ParseDuration("5s")
```

Comment:
change `time.ParseDuration("5s")` to `time.Second * 5` - no need to parse a string value when you know what its going to be. Using time.Second will provide a duration.

Reply:
You cannot assign time.Second to time.Duration and I haven't been able to find an example of how to convert seconds to duration.

Reply:
This also consistent with the env config value.

Reply:
So you can do this `defaultTimeout := time.Duration(5 * time.Second)` and then you wouldn't have to handle an error if unable to parse string